docs/imviz/plugins.rst

```diff
@@ -258,6 +260,12 @@ The columns are as follow:
 Once you have the results in a table, you can further manipulated them as
 documented in :ref:`astropy:astropy-table`.
 
+If you opted to fit a `~astropy.modeling.functional_models.Gaussian1D`
+to the radial profile, the last fitted model can be obtained by as follows.
+See :ref:`astropy:astropy-modeling` on how to manipulate the model::
+
+    my_gaussian1d = imviz.app.fitted_models['phot_radial_profile']
```

Comment:
What might be Cubeviz but none of that is even planned right now. Ideas welcome though. I am just trying to hook into existing machinery instead of creating even more API.